acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/ValueTransferWithAzureAcceptanceTest.java

```diff
@@ -64,6 +67,18 @@ public static void setUpBase() {
     ethSigner.awaitStartupCompletion();
   }
 
+  static void tearDownBase() {
```

Comment:
You can also use @AfterAll if you want to turn them off after test case is unloaded .... Probably also try/catch around each shutdown so that you catch and log exception instead of bubbling up.

Reply:
So have added @AfterAll as well as the shutdown hook, as that appears to be what we've done in a few places.
Have put exception handling around the shutdown - to ensure both items are attempted to be shutdown - but have let the exception percolate upwards, we probably want to see any failures rather than eating them.